tests/baselines/reference/capturedParametersInInitializers2.js

```diff
@@ -19,11 +19,13 @@ function foo(y, x) {
     var _a;
 }
 function foo2(y, x) {
-    if (y === void 0) { y = /** @class */ (function () {
-        function class_2() {
-            this[x] = x;
-        }
-        return class_2;
-    }()); }
+    if (y === void 0) { y = (_a = /** @class */ (function () {
+            function class_2() {
+                this[_b] = x;
+            }
+            return class_2;
+        }()), _b = x,

```

Comment:
Should `_b = x,` be on its own line?

Reply:
I'm more befuddled as to why `_a` appears on the following line, actually (because they should all be inline). AFAIK the comma list doesn't specify multiline (nor should it), so I'm not sure where the break is coming from.

Reply:
Then again statics appear on their own lines in the same context, apparently, so I suppose so?